infinispan-embedded/src/main/java/org/hibernate/ogm/datastore/infinispan/impl/InfinispanEmbeddedStoredProceduresManager.java

```diff
@@ -59,7 +55,7 @@
 		Callable<?> callable = instantiate( storedProcedureName, className, classLoaderService );
```

Comment:
`callable` doesn't say much about he content of the variable, I assume this is the `storedProcedure` isn't it?

Reply:
It seems it is not something from this PR, so never mind